SparkPost/2web2ui
# SA-2226 - (UI) Add JSDoc comments to *all* helper functions that Analytics Report leverages

[SA-2226](https://sparkpost.atlassian.net/browse/SA-2226)

### What Changed

- Adds JSDoc comments for all helper functions leveraged by Analytics Report

### How To Test

- Verify all helpers leveraged by the feature are properly documented
- Verify that the documentation is correct and without mistakes

### To Do
